packages/peregrine/lib/talons/CreateAccount/useCreateAccount.js

```diff
@@ -96,7 +96,9 @@ export const useCreateAccount = props => {
                 await removeCart();
 
                 // Delete stale cart data from apollo
-                await deleteCacheEntry(apolloClient, key => key.match(/^Cart/));
+                await deleteCacheEntry(apolloClient, key =>
+                    key.match(/^\$?Cart/)
+                );
```

Comment:
Consider extracting this function into a utility, since we're doing this three times in this PR alone. Perhaps `deleteStaleCartData`?

Reply:
9deb452e0